### PR TITLE
Issue #34: use system variable to customize path patch

### PR DESCRIPTION
**Pull request by HuGanghui**
Issue #34: use system variable to customize path patch

Need to modify `patch-filter-jar` in contribution repo: add to `<propertyExpansion>checkstyle.pathfilter.patch=${checkstyle.pathfilter.patch.from.env}</propertyExpansion>` to `contribution/checkstyle-tester/pom.xml`

```xml
<plugin>
       <groupId>org.apache.maven.plugins</groupId>
	<artifactId>maven-checkstyle-plugin</artifactId>
	<version>${checkstyle-plugin.version}</version>
	<!-- 	Specifying configuration here will take effect on the execution of "mvn site",
			but will not take effect on the execution of "mvn checkstyle:checkstyle"  -->
	<configuration>
	         <enableFilesSummary>false</enableFilesSummary>
	         <failsOnError>${checkstyle.failsOnError}</failsOnError>
                 <propertyExpansion>checkstyle.pathfilter.patch=${checkstyle.pathfilter.patch.from.env}</propertyExpansion>
         </configuration>
</plugin>
```

### CODE REVIEW

File: src/main/resources/patchConfig.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
     </module>
 
     <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchFilter">
-        <property name="file" value="/Users/hgh/Documents/GitHub/patch-filters/DiffReport/patch.txt"/>
+        <property name="file" value="${checkstyle.pathfilter.patch}"/>
```
Comment (rdiachenko) [inline src/main/resources/patchConfig.xml:21]: Update to `${checkstyle.patchfilter.patch}` (`c` is missed in `pathfilter` word)

File: README.md [hunk 2]
```diff
@@ -48,10 +48,9 @@ refer to [here](https://github.com/checkstyle/checkstyle/blob/ec4d06712ab203d31d
 you can skip Translation Check and checks that reuqires extra config file (Header and RegExpHeader), patchfilter setting is like:
 ```xml
 <module name="com.github.checkstyle.patchfilter.SuppressionPatchFilter">
```
Comment (rdiachenko) [inline README.md:50]: `com.puppycrawl.tools.checkstyle.filters.SuppressionPatchFilter`

File: README.md [hunk 3]
```diff
@@ -78,6 +77,10 @@ path/to/patchConfig.xml
 # Attention: num should be greater than 1, because if num is 1, no report will be created.
 4
 ```
+then, add Environment variables:
+```bash
+checkstyle.pathfilter.patch.from.env=/path/to/patch-filters/DiffReport/patch.txt
```
Comment (rdiachenko) [inline README.md:82]: Use `checkstyle.patchfilter.patch` for env variable name. Check my PR - https://github.com/checkstyle/contribution/pull/491, same names should be used in the current PR.

Comment (HuGanghui) [reply README.md:82]: done

File: README.md (hunk 2 above)
Comment (HuGanghui) [reply README.md:50]: done

File: src/main/resources/patchConfig.xml (hunk 1 above)
Comment (HuGanghui) [reply src/main/resources/patchConfig.xml:21]: done

**Pull request merged**